readme.md

```diff
@@ -17,24 +17,36 @@ This will generate png image using black and white and default font
 
 ![barcode](http://q.i-systems.pl/file/fa869375.png "Generated barcode")
 
- 
+
 Custom barcode:
 
 ```php
 $builder = new Barcode\Builder();
 $builder->setBarcodeType('gs1-128');
 $builder->setFilename('barcode.png');
+$builder->saveImageFile(false);

```

Comment:
Thanks for merging man. Btw I forgot to remove this line from readme, sorry. :sweat_smile: